CONTRIBUTING.md

```diff
@@ -153,6 +153,8 @@ each link to read more about their rules.
 -   [Quine Rules][9]
 -   [Reverse a String Rules][8]
 -   [Roman Numeral Conversion][26]
```

Comment:
While you're here, can you update these last three to include the word `Rules` at the end like the rest of the list?